### DISCUSSION TITLE
Is this repository still maintained?

### DISCUSSION THREAD

**Discussion by Timothy-Liuxf** [rahuldkjain/github-profile-readme-generator#793] [Q&A] [answered]
This repository hasn't been updated for more than one year, since the last commit is [888aff31e](https://github.com/rahuldkjain/github-profile-readme-generator/commit/888aff31e1d26dd2a6acf6afebbc34970aeb0118).
Then I wanna know if this repo is still maintained by the owner.
I think that this repo is a pretty good project and I love this repo very much, and that it is a great pity if this repo is no more maintained.

**Comment by rahuldkjain** [comment, marked as answer]
Hey @Timothy-Liuxf, I am glad you liked the tool. I was going through a lot of things lately. Failed to maintain the project gracefully. Now things are looking fine and I am thinking of revamping this project with a better tech stack. Will keep things posted now.

**Comment by Timothy-Liuxf** [comment]
Thank you very much. Looking forward to a better project, and sincerely hope this project becomes more and more popular! 🎉🎉🎉

**Comment by hariom-dsde** [comment]
yes someone
